src/query/storages/common/txn/src/lib.rs

```diff
@@ -34,10 +38,16 @@ pub enum TxnState {
     Fail,
 }
 
-#[derive(Debug, Clone)]
-struct TxnBuffer {
-    mutated_tables: HashMap<String, (UpdateTableMetaReq, TableInfo)>,
-    stream_tables: HashMap<String, StreamSnapshot>,
+#[derive(Debug, Clone, Default)]
```

Comment:
Better to spill out all the `struct` and `function` in the `lib.rs` to single file, make the `lib.rs` only for export and clean.